internal/jstest/jstest.go

```diff
@@ -29,7 +31,28 @@ func Run(ctx context.Context, args []string) (*internal.TestResults, error) {
 
 	ts := &internal.TestResults{}
 	ts.TestResults = make(map[string]internal.TestResult)
-	for _, testName := range args {
+
+	files := []string{}
+
+	for _, f := range args {
+		dir, err := os.Getwd()
+		if err != nil {
+			return nil, err
+		}
+
+		matches, err := filepath.Glob(filepath.Join(dir, f))
```

Comment:
```suggestion
		matches, err := filepath.Glob(dir, f)
```

The current directory is implied

Reply:
Fixed.